vue-app/src/views/TransactionSuccess.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="container">
+    <div class="image-wrapper">
+      <img class="docking" src="@/assets/docking.png" />
+      <img class="money" src="@/assets/money.gif" />
+      <img class="money" src="@/assets/confetti.gif" />
+    </div>
+    <div class="content">
+      <span class="contributed-icon">🎉</span>
+      <p
+        v-if="$route.params.type === 'reallocation'"
+        class="contributed-header"
+      >
+        {{ formatContribution() }}
+        {{ $store.state.currentRound.nativeTokenSymbol }} reallocated!
```

Comment:
You could just use `currentRound.nativeTokenSymbol` here since you have defined `get currentRound()`.